third_party/ssc/ssc/core.cpp

```diff
@@ -64,9 +64,9 @@ bool compute_module::compute(handler_interface *handler, var_table *data) {
     try { // catch any 'general_error' that can be thrown during precheck, exec, and postcheck
 
         if (!evaluate()) return false;    // This can be enabled when we want automatic updating of interdependent-inputs
-        if (!verify("precheck input", SSC_INPUT)) return false;
+        // if (!verify("precheck input", SSC_INPUT)) return false;
         exec();
-        if (!verify("postcheck output", SSC_OUTPUT)) return false;
+        // if (!verify("postcheck output", SSC_OUTPUT)) return false;
```

Comment:
The "compute module" (cmod) in SSC verifies all inputs at each timestep are within valid ranges or enums. We're doing that validation with the idd, so no need to redo that here.